### DISCUSSION TITLE
Period insert plugin updated to support Kimai 2.0.

### DISCUSSION THREAD

**Discussion by lnNgyn1** [kimai/kimai#4550] [Plugins]
Hi Kevin, I managed to update the period insert plugin by MR Software GmbH to support Kimai 2.0, and I wanted to share it for others to use. What steps do I need to take? I currently have it on a GitLab repository. Thanks

**Comment by kevinpapst** [comment]
Do you have the permissions to do that?

**Reply by lnNgyn1** [reply]
Good question, I updated it for my company since they used it on a previous version of Kimai, but I haven't asked the original creator yet. So I don't have the permissions from MR Software GmbH right now. I did acknowledge them on the README of the repository, and I can add more acknowledgements elsewhere. I guess I'll send them an email first but since they closed down I don't know if it will reach them.

**Reply by kevinpapst** [reply]
Yes please, do that. And I try to reach the previous owner as well.

**Reply by lnNgyn1** [reply]
OK, I just received an email from Johannes Roth at MR Software GmbH, and he gave me permission to publish it for free.

**Reply by kevinpapst** [reply]
Awesome 👍 

And I can confirm that, as he told me the same today. Thanks @j0hannesr0th ❤️ 

 Do you want to create the necessary pages in the store?

**Reply by lnNgyn1** [reply]
Sure, I can do that. The plugin is pretty simple to explain. I'll publish it on GitHub and share it with you first.

EDIT: The repository is linked [here](https://github.com/lnNgyn1/PeriodInsertBundle).
